pkg/api/sort.go

```diff
@@ -357,7 +356,7 @@ func sortByName[T orderedWithName](direction string, results []T) {
 	})
 }
 
-func sortByFunc[T orderedWithName, U constraints.Ordered](direction string, results []T, fn func(int) U) {
+func sortByFunc[T orderedWithName, U cmp.Ordered](direction string, results []T, fn func(int) U) {
```

Comment:
The `constraints.Ordered` from `golang.org/x/exp/constraints` is also included into Go 1.21 as `cmp.Ordered`. See https://pkg.go.dev/cmp#Ordered